test/Microsoft.DotNet.Tools.Tests.Utilities/Assertions/FileInfoAssertions.cs

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.IO;
+using FluentAssertions;
+using FluentAssertions.Execution;
+
+namespace Microsoft.DotNet.Tools.Test.Utilities
+{
+    public class FileInfoAssertions : FileSystemInfoAssertions<FileInfo, FileInfoAssertions>
+    {
+        public FileInfoAssertions(FileInfo subject)
+        {
+            Subject = subject;
+        }
+
+        public AndConstraint<FileInfoAssertions> ContainText(string expectedText, string because = "", params object[] becauseArgs)
+        {
+            var text = Subject.OpenText().ReadToEnd();
+
+            Execute.Assertion
+                .ForCondition(text.Contains(expectedText))
```

Comment:
Duplication. It just has an inverted condition and a slightly different error message.